scenarios/12_Leaving_Raedwood.cfg

```diff
@@ -262,6 +280,43 @@ Move Ruvio to signpost to ask him for advice."}
         {INTERROGATION12}
     [/event]
 
+    [event]
+        name=last breath
+        [filter]
+            id=Quivre O Raedbor
+        [/filter]
+        [filter_second]
+            id=Gawen Hagarthen
+        [/filter_second]
+        {MSGW_Ruvio _"Gawen! I thought I told you not to waste your time with that one!"}
```

Comment:
Doesn't really sound like Ruvio. He's not so demanding of Gawen, seems more mentor like. I wonder if you can rewrite this line?

Reply:
How about rephrasing as a question, e.g. "Gawen, didn't I tell you not to waste your time with that one?"

Reply:
> He had nothing to say after all.

Reply:
ok, I used that in 6f04548